_source/_data/toc.yml

```diff
@@ -71,9 +71,9 @@ firstLevel:
   - title: Infrastructure Monitoring
     url: /user-guide/infrastructure-monitoring/
     thirdLevel:
-    - title: Data rollups
-      url: /user-guide/infrastructure-monitoring/data-rollups.html
-    - title: Configure Grafana drilldown links
```

Comment:
@imnotashrimp  - This slipped into the master by mistake. Needs to be erased here and added from the right PR.